### PR TITLE
Began Assignment 7

### PR DESCRIPTION
**Pull request by TatyanaHubbard**
Enabled Nullable, Wrote Sort Function, and Started Tests

### CODE REVIEW

File: DelegateFun/Sorter/SortUtility.cs [hunk 1]
```diff
@@ -1,10 +1,42 @@
 ﻿using System;
+using System.Reflection;
 
 namespace Sorter
 {
     public class SortUtility
     {
         // Sort method should be implemented here
         // It should accept an int[] and a delegate you define that performs the actual comparison
```
Comment (dnisyndrum) [inline DelegateFun/Sorter/SortUtility.cs]: Every thing looks good!
Just me being knit picky a little, don't need the comments from the blank assignment 7.

File: DelegateFun/Sorter.Tests/SortUtilityTests.cs [hunk 2]
```diff
@@ -8,6 +11,48 @@ public class SortUtilityTests
...
+            int[] orderArr = { 1, 2, 3, 4, 5, 6, 7, 8, 9 };
+
+            SortUtility sortUtility = new SortUtility();
+            arr = sortUtility.Sort(arr, (int first, int second) => { return first < second; });
+
+            Assert.IsTrue(arr.SequenceEqual(orderArr));
+        }
+
+        [TestMethod]
+        public void SortUtility_ShouldSortAscending_UsingALambdaExpression()
```
Comment (JerettLatimer) [inline DelegateFun/Sorter.Tests/SortUtilityTests.cs]: All of your tests sort the data in an ascending order. The assignment specifications require at least three different orderings of the data. In my solution, I did ascending, descending and alphabetical descending.

File: DelegateFun/Sorter.Tests/SortUtilityTests.cs [hunk 3]
```diff
@@ -8,6 +11,48 @@ public class SortUtilityTests
...
+
+            SortUtility sortUtility = new SortUtility();
+            arr = sortUtility.Sort(arr, (int first, int second) => first < second);
+
+            Assert.IsTrue(arr.SequenceEqual(orderArr));
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void SortUtility_ShouldSortAscending_PassingNull()
```
Comment (JerettLatimer) [inline DelegateFun/Sorter.Tests/SortUtilityTests.cs]: Don't forget to also create a test that passes null for your sorting function!

File: DelegateFun/Sorter.Tests/Sorter.Tests.csproj [hunk 4]
```diff
@@ -1,8 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.0</TargetFramework>
-
+    <Nullable>enable</Nullable>
```
Comment (JerettLatimer) [inline DelegateFun/Sorter.Tests/Sorter.Tests.csproj]: Don't forget to enable Nullability for the SorterUtility project as well. Also, add code analysis to both projects!

File: DelegateFun/Sorter.Tests/SortUtilityTests.cs (hunk 2 above)
Comment (breaman) [reply DelegateFun/Sorter.Tests/SortUtilityTests.cs]: This is correct, the tests should not have sorted the same way every time.
